data/language/en-GB.txt

```diff
@@ -3661,6 +3661,15 @@ STR_6406    :GOG RollerCoaster Tycoon 2 Installer
 STR_6407    :This may take a few minutes.
 STR_6408    :Please install “innoextract” to extract GOG Installer, then restart OpenRCT2.
 STR_6409    :The selected file is not the offline GOG Installer for RollerCoaster Tycoon 2. You may have downloaded the GOG Galaxy downloader stub or selected the wrong file.
+STR_6410    :{WINDOW_COLOUR_2}Your Progress:
+STR_6411    :{BLACK}-
+STR_6412    :{BLACK}Guests: {COMMA32} of {COMMA16}
```

Comment:
Different format types used here, should both be `COMMA32`

Reply:
This is correct, according to the code. The gNumGuestsInPark variable is a uint32_t and the Objective NumGuests variable is a uint16_t. I was confused about that, too.

Reply:
Okay, the objective num guests will at some point become a `uint32_t`, so maybe you can cast it to int32 for now.

Reply:
I've converted to uint32_t for now. I doubt that there'll ever be a negative number of guests and converting it to int32 would add information that isn't present in the base type.

Reply:
@IntelOrca 
Is this ok?